src/utils/AddNewTodo.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+interface Props {
+  onTodoAdded: (title: string) => void;
+  isLoading: boolean;
+}
+
+export const AddNewTodo: React.FC<Props> = ({ onTodoAdded, isLoading }) => {
```

Comment:
```suggestion
export const AddTodoForm: React.FC<Props> = ({ onTodoAdded, isLoading }) => {
```
move it from utils - it's component